src/scriptCache.ts

```diff
@@ -94,3 +94,59 @@ export function removeScriptUpdater(
     (currentUpdater) => currentUpdater !== updater
   );
 }
+
```

Comment:
Not your changes, but do you want to fix the spelling above `getCahcedScriptUpdaters`?